src/modules/class.js

```diff
@@ -0,0 +1,24 @@
+export const list = JSON.parse(localStorage.getItem('list')) || [];
+
+export class ToDoTasks {
+  constructor(description = '', completed = false, index = null) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+export const toDoTasks = new ToDoTasks();
+
+export const updateLocalStorage = () => {
+  const newLists = document.querySelectorAll('li');
+  const list = [];
+  newLists.forEach((newList) => {
+    list.push({
+      description: toDoTasks.description,
+      completed: newList.classList.contains('checked'),
+      index: list.length,
```

Comment:
- [ ] From the requirements of this project, if the 2nd task is added to the list, the index of that task should be equal to 2. This is not the same behavior in your app as you can see in the screenshot below:

![Screenshot from 2022-05-11 00-30-23](https://user-images.githubusercontent.com/63087491/167741315-5b90032b-770e-4c66-b12b-9fafac737600.png)

Please kindly fix this issue :pray:

Reply:
Fixed